### PR TITLE
[LMIDist] Add test that calls `LmiDistRollingBatch` with python script

### PR DESCRIPTION
**Pull request by KexinFeng**
## Description ##

This adds a unit test that calls `LmiDistRollingBatch` with python script, for the following purpose:
1. Easiness of further development, e.g. speculative decoding
2. Use the greedy output text as benchmark, to prevent possible text quality degradation from model level modification
3. Easy demonstration and benchmarking
4. Easy debugging

## Next step ##
In the next step, this test can be added to integration test: .github/workflows/rolling_batch_integration.yml
which will require writing a new shell script for running `python3 file` in docker.

### CODE REVIEW

File: engines/python/setup/djl_python/tests/test_lmi_dist.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
+import unittest
```
Comment (frankfliu) [inline engines/python/setup/djl_python/tests/test_lmi_dist.py:1]: Please add license header.

File: engines/python/setup/djl_python/tests/test_lmi_dist.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+import torch
+import os, sys
+
+# script_directory = os.path.dirname(os.path.abspath(__file__))
```
Comment (frankfliu) [inline engines/python/setup/djl_python/tests/test_lmi_dist.py:6]: remove commented out code

File: engines/python/setup/djl_python/tests/test_lmi_dist.py [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+            # TODO: fix this. weight model.layers.0.self_attn.rotary_emb.inv_freq does not exist
+            # "TheBloke/Llama-2-7B-Chat-AWQ",
+            "TinyLlama/TinyLlama-1.1B-Chat-v0.6",
+            # TODO: fix this. weight model.layers.0.self_attn.rotary_emb.inv_freq does not exist
+            # "TinyLlama/TinyLlama-1.1B-python-v0.1",
+            # g5.12xlarge single gpu ok. But no way to clear the gpu memory after running llama-2-7b thus cause OOM
+            # "codellama/CodeLlama-7b-hf"
+        ]
+        expected_text_30 = {
+            "TheBloke/Llama-2-7B-Chat-fp16": {
```
Comment (lanking520) [inline engines/python/setup/djl_python/tests/test_lmi_dist.py:47]: let's remove this then

Comment (lanking520) [reply engines/python/setup/djl_python/tests/test_lmi_dist.py:47]: But it is just too big to test on a CPU machine. We could use a better or smaller model for evaluation and testing

**Review by lanking520 — APPROVED** [review]

**Pull request merged**